src/pages/docs/infrastructure/accounts/aws/index.md

```diff
@@ -33,18 +33,35 @@ See the [AWS documentation](https://oc.to/aws-access-keys) for instructions to c
 Support for OpenID Connect authentication to AWS requires Octopus Server version 2024.1
 :::
 
+To use OpenID Connect authentication you have to follow the [required minimum configuration](/docs/infrastructure/accounts/openid-connect#configuration).
+
 See the [AWS documentation](https://oc.to/aws-oidc) for instructions to configure an OpenID Connect identity provider.
 
+:::div{.info}**If using the AWS CLI or API to configure the identity provider.**
+
+See the [AWS Documentation](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_create_oidc_verify-thumbprint.html) for instructions on how to obtain the thumbprint of your Octopus Server.
+:::
+
 When setting up the identity provider you need to use the host domain name of your server as the **Audience** value, as configured under **Configuration->Nodes->Server Uri**.
 
-To use OpenID Connect authentication you have to follow the [required minimum configuration](/docs/infrastructure/accounts/openid-connect#configuration).
+#### Configuring AWS OIDC Account
 
 1. Navigate to **Infrastructure ➜ Accounts**, click the **ADD ACCOUNT** and select **AWS Account**.
-1. Add a memorable name for the account.
-1. Provide a description for the account.
-1. Set the Role ARN to the ARN from the identity provider associated role.
-1. Set the Session Duration to the Maximum session duration from the role, in seconds.
-1. Click the **SAVE AND TEST** to save the account and verify the credentials are valid.
+2. Add a memorable name for the account.
+3. Provide a description for the account.
+4. Set the **Role ARN** to the ARN from the identity provider associated role.
+5. Set the **Session Duration** to the Maximum session duration from the role, in seconds.
+6. Click **SAVE** to save the account.
+7. Before you can test the account you need to add a condition to the identity provider in AWS:
```

Comment:
This is in the _IAM -> Roles -> Trust Relationships_ right? I think spelling that out here is worthwhile, people might just be looking under the Identityprovider

Reply:
Should we also include the Principal to better understand the whole trust relationship JSON? 

I'm also not sure if it's worth mentioning that they can avoid the subject requirement, from a security perspective probably better to leave that out but during testing I configured mine on just the audience and that was pretty helpful in my case.

`
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Effect": "Allow",
            "Principal": {
                "Federated": "arn:aws:iam::720766170633:oidc-provider/a333-2001-8003-ac03-7500-6dfc-aa25-5c69-abec.ngrok-free.app"
            },
            "Action": "sts:AssumeRoleWithWebIdentity",
            "Condition": {
                "StringEquals": {
                    "a333-2001-8003-ac03-7500-6dfc-aa25-5c69-abec.ngrok-free.app:aud": "a333-2001-8003-ac03-7500-6dfc-aa25-5c69-abec.ngrok-free.app"
                }
            }
        }
    ]
}
`

Reply:
> not sure if it's worth mentioning that they can avoid the subject requirement
I'd rather not, the chance that someone then forgets to update it to after testing and leaving their identity provider wide open is higher than I'd like compared to the convenience.